src/Parser/Rogue/Subtlety/CombatLogParser.js

```diff
@@ -1,110 +1,22 @@
-import React from 'react';
-
-import Icon from 'common/Icon';
 import CoreCombatLogParser from 'Parser/Core/CombatLogParser';
-import SPELLS from 'common/SPELLS';
-import SpellIcon from 'common/SpellIcon';
-import StatisticBox from 'Main/StatisticBox';
-
-import ShadowDance from './Modules/Features/ShadowDance';
+import DamageDone from 'Parser/Core/Modules/DamageDone';
 import CastEfficiency from './Modules/Features/CastEfficiency';
 
-function formatThousands(number) {
-  return (`${Math.round(number || 0)}`).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
-}
-
-function formatNumber(number) {
-  if (number > 1000000) {
-    return `${(number / 1000000).toFixed(2)}m`;
-  }
-  if (number > 10000) {
-    return `${Math.round(number / 1000)}k`;
-  }
-  return formatThousands(number);
-}
-
-function formatPercentage(percentage) {
-  return (Math.round((percentage || 0) * 10000) / 100).toFixed(2);
-}
+import { ShadowBladesUptime, ShadowDance, DarkShadowContribution, DarkShadowEvis, SymbolsOfDeathUptime } from './Modules';
```

Comment:
Rather than doing this import using an index.js in Modules, current style is to just import one per line, and skip the index.

I'm guessing the point of doing it this way is so you don't have to change a lot of imports if you move one of the modules? It's probably not a big deal you're doing it this way, but should talk to @MartijnHols about it and see what he thinks.

Reply:
I contacted Martijn, its just the way I've done it before and I don't really have a strong opinion on using it here or not.

Reply:
I'd prefer keeping it consistent across all specs unless we're going to deprecate one method in favor of another. Having alternative solutions to simple problems would make it harder for us to make cross-spec improvements.

I think it's a good idea, I've been thinking about if we could apply it everywhere but there are several problems, and I don't think the variances I considered solve those either.
I don't want to have to edit 2 files instead of 1 to enable modules for no real tangible gain. I was also thinking about returning a default from the Modules index.js file like the index file in the SPELLS and ITEMS directories and other locaitons, but that would mean we have 2 different approach for core and spec-specific modules and it gets harder with the alternative when there are options (`damageDone: [DamageDone, { showStatistic: true }],`). Another alternative would have been to just move the entire specModules into the index.js but I don't think it makes things easier to use.

As for the rest I trust @kfinch his judgment.

Reply:
Alright, removed for now.